### PR TITLE
Created README and added documentation for missing features

### PR DESCRIPTION
**Pull request by MatthewJohn**

### CODE REVIEW

File: Documentation/Cluster.rst [hunk 1]
```diff
@@ -69,3 +71,68 @@ To the remove a node from the cluster, run:
     
     sudo mcvirt cluster remove-node --node <Remote Node Name>
     
+
+Get Cluster information
+-----------------------
+
+* In order to view status information about the cluster, use the 'info' parameter for MCVirt, without specifying a VM name:
```
Comment (ndyer) [inline Documentation/Cluster.rst]: Use the double colon in line at the end of the paragraph, see https://en.wikipedia.org/wiki/ReStructuredText#Literal_blocks

same elsewhere

File: Documentation/ControllingVMs.rst [hunk 2]
```diff
@@ -170,23 +170,26 @@ Monitoring Resources
...
+  ::
 
-  * Power off the VM
+    sudo mcvirt backup --create-snapshot --disk-id <Disk ID> <VM Name>
 
-  * Login to <Node> as root
+2. The returned path provides access to the disk at the time that the snapshot was created
 
-  * Ensure the LV of the HDD is active
+**Warning:** The snapshot is 500MB in size, meaning that once the VM has changed 500MB of space on the disk, the VM will no longer be able to write to it's disk
```
Comment (ndyer) [inline Documentation/ControllingVMs.rst]: its

File: Documentation/ModifyingVMs.rst [hunk 3]
```diff
@@ -91,4 +90,29 @@ Attaching ISO
     sudo mcvirt start <VM Name> --iso <Name of ISO file>
     
 
-* The VM must be stored within /var/lib/mcvirt/iso and specifying just the filename in the above command.
\ No newline at end of file
+* The ISO file must be stored within /var/lib/mcvirt/iso and specifying just the filename in the above command.
```
Comment (ndyer) [inline Documentation/ModifyingVMs.rst]: Funny english. It looks like you split a sentence.

File: Documentation/Cluster.rst [hunk 4]
```diff
@@ -69,3 +71,60 @@ To the remove a node from the cluster, run:
...
+DRBD verification
+-----------------
+
+MCVirt has the ability to start/monitor DRBD verifications (See the `DRBD documentation <https://drbd.linbit.com/users-guide/s-use-online-verify.html>`_.
+
+The verification can be performed by using::
+
+    sudo mcvirt verify <--all>|<VM Name>
+
+This will perform a verification of the specified (or all of the DRBD-backed VMs, if the '--all' is specified). Once the verification is complete, an exception is thrown if any of the verifications fail.
```
Comment (ndyer) [inline Documentation/Cluster.rst]: "if the --all" is a bit weird

File: Documentation/ModifyingVMs.rst [hunk 5]
```diff
@@ -82,13 +82,28 @@ Add/Remove Network Adapter
...
 
-* The VM must be stored within /var/lib/mcvirt/iso and specifying just the filename in the above command.
\ No newline at end of file
+* The ISO file must be stored within /var/lib/mcvirt/iso.
+
+
+VM Locking
+----------
+
+VMs can be locked by superusers, which stops them from being started, stopped and migrated
```
Comment (ndyer) [inline Documentation/ModifyingVMs.rst]: "or migrated" ?

**Comment by MatthewJohn** [comment]
Signed off by @ndyer and @s-knibbs

**Pull request merged**